composer.json

```diff
@@ -29,5 +29,10 @@
         "psr-4": {
             "OpenTracingTests\\": "./tests/"
         }
+    },
+    "scripts": {
+        "test": "php ./vendor/bin/phpunit tests",
+        "check-linting": "./vendor/bin/phpcs --standard=ZEND --standard=PSR2 --ignore=*/vendor/* ./",
```

Comment:
I am not a native, nor have clear understanding on how to use the `lint` word so I would go with them as well 👍